packages/charts/react-charting/src/components/DeclarativeChart/imageExporter.ts

```diff
@@ -43,15 +43,37 @@ function toSVG(chartContainer: HTMLElement, background: string) {
     throw new Error('SVG not found');
   }
 
-  const { width: svgWidth, height: svgHeight } = svg.getBoundingClientRect();
-  const classNames = new Set<string>();
-  const legendGroup = cloneLegendsToSVG(chartContainer, svgWidth, svgHeight, classNames);
-  const w1 = Math.max(svgWidth, legendGroup.width);
-  const h1 = svgHeight + legendGroup.height;
   const clonedSvg = d3Select(svg.cloneNode(true) as SVGSVGElement)
     .attr('width', null)
     .attr('height', null)
     .attr('viewBox', null);
+  const svgElements = svg.getElementsByTagName('*');
+  const clonedSvgElements = clonedSvg.node()!.getElementsByTagName('*');
+
+  for (let i = 1; i < svgElements.length; i++) {
+    const properties = [
```

Comment:
>properties [](http://example.com/codeflow?start=10&length=10)

This approach looks fine. Lets proceed with this.

Reply:
okay